OrbitGl/GpuTrack.cpp

```diff
@@ -347,3 +354,53 @@ std::vector<std::shared_ptr<TimerChain>> GpuTrack::GetAllChains() {
   }
   return chains;
 }
+
+//-----------------------------------------------------------------------------
+std::string GpuTrack::GetBoxTooltip(PickingID id) const {
+  TextBox* text_box = time_graph_->GetBatcher().GetTextBox(id);
+  if (!text_box || text_box->GetTimer().m_Type == Timer::CORE_ACTIVITY) {
+    return "";
+  }
+
+  std::string gpu_stage =
+      string_manager_->Get(text_box->GetTimer().m_UserData[0]).value_or("");
+  if (gpu_stage == kSwQueueString) {
+    return GetSwQueueTooltip(text_box->GetTimer());
+  } else if (gpu_stage == kHwQueueString) {
+    return GetHwQueueTooltip(text_box->GetTimer());
+  } else if (gpu_stage == kHwExecutionString) {
+    return GetHwExecutionTooltip(text_box->GetTimer());
+  }
+
+  return "";
+}
+
+std::string GpuTrack::GetSwQueueTooltip(const Timer& timer) const { 
+  return absl::StrFormat(
+    "<b>Software Queue</b><br/>"
+    "<i>Time between amdgpu_cs_ioctl and amdgpu_sched_run_job.</i>"
```

Comment:
Can be done later, but we may want to add an explanation for amdgpu_cs_ioctl and amdgpu_sched_run_job. 

amdgpu_cs_ioctl = job submission from user space
amdgpu_sched_run_job = job scheduled by the driver

Reply:
I think that makes sense, I added a bit of a shortened version ("job submitted" and "job scheduled") to keep the tooltip a bit shorter, wdyt?